dot-net-sdk/EppoClientConfig.cs

```diff
@@ -1,11 +1,32 @@
 ﻿using eppo_sdk.constants;
+using eppo_sdk.dto;
+using eppo_sdk.dto.bandit;
 using eppo_sdk.logger;
 
-namespace eppo_sdk
+namespace eppo_sdk;
+public class EppoClientConfig
 {
-    public record EppoClientConfig(string ApiKey,
-        IAssignmentLogger AssignmentLogger)
+    public string ApiKey;
+    public readonly IAssignmentLogger AssignmentLogger;
+
+    public EppoClientConfig(string apiKey,
+                            IAssignmentLogger? assignmentLogger)
     {
-        public string BaseUrl { get; set; } = Constants.DEFAULT_BASE_URL;
+        this.ApiKey = apiKey;
+        this.AssignmentLogger = assignmentLogger ?? new DefaultLogger();
```

Comment:
Refactored here to allow assignmentLogger param to be nullable and set a noop logger in its place.